test/DisableTest.php

```diff
@@ -30,15 +33,15 @@ class DisableTest extends TestCase
 
     public function setUp()
     {
-        $this->projectDir = vfsStream::setup('project', null, [
+        $this->projectDir = vfsStream::setup($this->projectDirName, null, [
```

Comment:
Why are you changing all the tests by adding by moving the `project` into a property?